packages/common/src/constants.js

```diff
@@ -183,6 +183,9 @@ const USER_ROLES = [
   'Grantee Specialist',
   'Health Specialist',
   'System Specialist',
+  'Interim Management Team',
+  'Integrated Service Team',
+  'Grants Management Specialist',
```

Comment:
It is a real bummer that we maintain this list and a list in the database. Do we need both?

Reply:
agree.